Directory.Build.props

```diff
@@ -138,6 +138,10 @@
 
     <_runtimeOS Condition="$(_runtimeOS.StartsWith('tizen'))">linux</_runtimeOS>
     <_runtimeOS Condition="'$(PortableBuild)' == 'true'">$(_portableOS)</_runtimeOS>
+
+    <_packageOS Condition="'$(CrossBuild)' == 'true'">$(_hostOS.ToLowerInvariant)</_packageOS>
```

Comment:
What is the difference between portableOS, runtimeOS and packageOS? I think it can use some comments.

Reply:
These are the intermediate private properties. I think we should document the four public properties (which are in PascalCasing, without underscore), and also explain the process of calculation that is inherently complex.